### ISSUE TITLE
Follower node Metadata error

### ISSUE THREAD

**Issue by snissn** [filecoin-project/lotus#10820]
Running this branch https://github.com/filecoin-project/lotus/tree/mikers/feat/cassandra-store when I run a follower node connected to yugadb through cassandra and run this rpc command:

```
 curl http://localhost:4234/rpc/v1 -X POST -H "Content-Type: application/json" --dat
a '{"method":"Filecoin.EthEstimateGas","params":[{"from":"0x08ca94cdD897F4382cb4E83Cb58e976ce06a7E68","to":"0x08ca94cdD897F4382cb4
E83Cb58e976ce06a7E68","value":"0x186a0"}],"id":1,"jsonrpc":"2.0"}'
```
I get this error:

```

2023-05-03T14:29:44.995-1000    WARN    rpc     go-jsonrpc@v0.2.3/handler.go:406        error in RPC call to 'Filecoin.EthEstimateGas': failed to estimate gas:
    github.com/filecoin-project/lotus/node/impl/full.(*EthModule).EthEstimateGas
        /home/mikers/dev/filecoin-project/lotus/node/impl/full/eth.go:941
  - CallWithGas failed:
    github.com/filecoin-project/lotus/node/impl/full.(*EthModule).applyMessage
        /home/mikers/dev/filecoin-project/lotus/node/impl/full/eth.go:909
  - call raw get actor:
    github.com/filecoin-project/lotus/chain/stmgr.(*StateManager).callInternal
        /home/mikers/dev/filecoin-project/lotus/chain/stmgr/call.go:193
  - resolution lookup failed (t410fbdfjjtoys72dqlfu5a6llduxntqgu7tizvs7jga):
    github.com/filecoin-project/lotus/chain/state.(*StateTree).GetActor
        /home/mikers/dev/filecoin-project/lotus/chain/state/statetree.go:372
  - resolve address t410fbdfjjtoys72dqlfu5a6llduxntqgu7tizvs7jga:
    github.com/filecoin-project/lotus/chain/state.(*StateTree).lookupIDinternal
        /home/mikers/dev/filecoin-project/lotus/chain/state/statetree.go:337
  - actor not found
 ```

the error is resolved when i copy over the ~150k metadata storage db into the follower node lotus working directory.

**Comment by snissn** [comment]
I did some digging, and identified that the issue is that the genesis block is not correctly set up to include the correct fvm version number.

[v0 ](https://github.com/filecoin-project/lotus/blob/master/chain/actors/builtin/init/v0.go) is called not [v11](https://github.com/filecoin-project/lotus/blob/master/chain/actors/builtin/init/v11.go)


I'm thinking we can resolve this using the metadata back up command to send a back up meta data from the chain sync node to the follower when setting up this cluster

**Comment by snissn** [comment]
secondly i'm trying to use 

./lotus daemon  --genesis=gen.car

but the calibnet genesis block car file is incorrect on our documentation:

https://docs.filecoin.io/networks/calibration/details/#genesis

says the ipfs hash is `QmY581cXXtNwHweiC69jECupu9EBx274huHjSgxPNv1zAAj` which has encoding problems

```
mikers@DESKTOP-DKHQ69U /m/e/y/d/metadata [1]> ipfs get QmY581cXXtNwHweiC69jECupu9EBx274huHjSgxPNv1zAAj
Error: invalid path "QmY581cXXtNwHweiC69jECupu9EBx274huHjSgxPNv1zAAj": selected encoding not supported
```
for main net i'm able to ipfs get the cid for the genesis file and get to a certain point with the follower node:
```
genesis in the repo is not the one expected by this version of Lotus!
```
I think this is the correct error for running a calibnet node on mainnet.

for mainnet instructions for starting follower node would be:


```
ipfs get QmavMCf95w2UMYGD1J5GpHcWBWXR2jTFYmtAkgeroMmpk1
mv QmavMCf95w2UMYGD1J5GpHcWBWXR2jTFYmtAkgeroMmpk1 genesis.car
mkdir -p /mnt/e/yugafollower
 LOTUS_PATH=/mnt/e/yugafollower/  FULLNODE_API_INFO=http://localhost:1234    LOTUS_CASSANDRA_UNIVERS
AL_STORE=localhost:9042 ~/dev/filecoin-project/lotus/lotus daemon --follower  --genesis=genesis.car
```

**Comment by snissn** [comment]
I see that calibnet.car is actually in the repo!

but this doesn't quite get me all the way there, i'm still missing something:

```
LOTUS_PATH=/mnt/e/yugafollower/  FULLNODE_API_INFO=http://localhost:1234    LOTUS_CASSANDRA_UNIVERS
AL_STORE=localhost:9042 ~/dev/filecoin-project/lotus/lotus daemon   --genesis=/home/mikers/dev/filecoin-project/lotus/build/genesis/calibnet.car
```

**Comment by snissn** [comment]
resolved with f418aae5c
